okta-hosted-login/README.md

```diff
@@ -32,11 +32,16 @@ Plug these values into the `mvn` commands used to start the application.
 cd okta-hosted-login
 mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default \
     -Dokta.oauth2.clientId={clientId} \
-    -Dokta.oauth2.clientSecret={clientSecret} \
-    -Dokta.oauth2.postLogoutRedirectUri={absoluteLogoutRedirectUri} # (optional) configure this property to enable SSO logout. 
+    -Dokta.oauth2.clientSecret={clientSecret}
 ```
 
-> **NOTE:** Putting secrets on the command line should ONLY be done for examples, do NOT do this in production. Instead update the projects `application.yml`
+> **NOTE:** Putting secrets on the command line should ONLY be done for examples, do NOT do this in production. Instead update the projects `application.yml` if you're using a closed source control system. Otherwise, we recommend you store them as environment variables. For example:
```

Comment:
I wouldn't add the "closed source" bit. Those secrets still _shouldn't_ be checked in, they should be managed by some automation process when used in a file format (but that's harder to explain, so maybe just push the ENV var route)

```suggestion
> **NOTE:** Putting secrets on the command line should ONLY be done for examples, do NOT do this in production. Instead, we recommend you store them as environment variables. For example:
```

Reply:
updated both readme.